src/Toaster/Toaster.tsx

```diff
@@ -71,12 +92,13 @@ export default class Toaster extends React.Component {
 
   getToasts = () => {
     const { children, transitionStyles, transitionEnterTimeout, transitionLeaveTimeout } = this.props;
-    return React.Children.map(children, (el, i) => {
+    return React.Children.map(children, (_el, i) => {
+      const el = _el as React.ReactElement<any>;
       return (
         <TransitionWrapper
           {...{ transitionStyles, transitionEnterTimeout, transitionLeaveTimeout }}
           style={this.getTranslationStyle(i)}
-          key={el.key}
+          key={el.key || undefined}
```

Comment:
this is stupid, but `TransitionWrapper` accepts `string | undefined`, while `el.key` is `string | null`...